include/deal.II/matrix_free/portable_matrix_free.templates.h

```diff
@@ -355,12 +359,12 @@ namespace Portable
       using TeamHandle = Kokkos::TeamPolicy<
         MemorySpace::Default::kokkos_space::execution_space>::member_type;
       using SharedView1D =
```

Comment:
Maybe rename this to
```suggestion
      using SharedViewValues =
```